README.md

```diff
@@ -10,6 +10,13 @@ This cookbook is used for installing the [Octopus Deploy](http://octopusdeploy.c
 This is pre release and there will be major changes to this before its final release.  The recipes for installation and configuration will be switched into resources so people can use the library easier. Once this is found stable it will be released as version 1.0.0, until this point lock down to any minor version that you use.
 
 
+## Attributes
+
+If you manage tentacle firewall though group policy leave 'manage_firewall' `false`
+If you want the cookbook to manage the firewall, set `true`
+
+    node['octopus']['tentacle']['instance']['manage_firewall'] = false
```

Comment:
I actually have pr #42 to remove attribute files from the cookbook just because they are not used.  Everything is  using a resource.  When this cookbook was first created we did use the attributes but now we do not